script.js

```diff
@@ -310,4 +310,30 @@ function formValidation(e) {
 // client side form validation
 
 const form = document.getElementById('contactform');
-form.addEventListener('submit', formValidation);
\ No newline at end of file
+form.addEventListener('submit', formValidation);
+
+// local storage
+const myForm = document.querySelector('#contactform');
+
+// saving form data to local storage
+myForm.addEventListener('input', () => {
+  const info = {
+    name: document.querySelector('#user-name').value,
+    email: document.querySelector('#email').value,
+    message: document.querySelector('#message').value,
+  };
+  localStorage.setItem('storedData', JSON.stringify(info));
+});
+
+// get data from local storage
+let userObject = JSON.parse(localStorage.getItem('storedData'));
+if (!userObject) {
+  userObject = {
+    name: '',
+    email: '',
+    message: '',
+  };
+  document.querySelector('#user-name').value = userObject.name;
+  document.querySelector('#email').value = userObject.email;
+  document.querySelector('#message').value = userObject.message;
```

Comment:
- According to the project requirements: `When the user loads the page, your code checks if there's any saved data in the local storage and if so the input fields are pre-filled with the data`.
Your project:
<img width="958" alt="fbfs" src="https://user-images.githubusercontent.com/73607512/218050131-78b9479b-0ce9-4c53-b5ee-7935bd78b65c.PNG">